build/variables.yml

```diff
@@ -69,9 +72,16 @@
   # Flutter version.
   FlutterVersion: '3.22.1'
 
+  # Docker version.
+  DockerVersion: '25.0.5'
+
+  # MobSF.
+  mobSfApiKey: '8181'
```

Comment:
This needs to be documented.

Reply:
documented

Reply:
It's not explained how this value is obtained and how it is supposed to work for other apps. Is 8181 good for all apps created with this template? There needs to be more explanation either here or in SecurityScan.md.

Reply:
done